src/main/java/frc/robot/RobotContainer.java

```diff
@@ -270,7 +267,7 @@ public void testInit(String choice) {
                 drivetrain.seedFieldRelative();
                 scoringSubsystem.setAction(ScoringAction.TUNING);
                 // spotless:off
-                controller.leftBumper()
+                triggers.get("testing")
```

Comment:
I think @redPlover is re-constructing the controller bindings in his PR for test modes. It would be good to have a test configuration JSON for stuff like this

For example, we have SingleController and we could have SingleControllerTest for test mode using a single controller